src/Faker/Core/Medical.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Core;
+
+use Faker\Extension;
+
+/**
+ * @experimental This class is experimental and does not fall under our BC promise
+ */
+final class Medical implements Extension\MedicalExtension
+{
+    /**
+     * @var string[]
+     */
+    private $bloodTypes = ['A', 'AB', 'B', 'O'];
+
+    /**
+     * @var string[]
+     */
+    private $bloodRhFactors = ['+', '-'];
+
+    /**
+     * @inheritDoc
+     */
```

Comment:
```suggestion
```

Can we remove this?

This information is automatically inherited, and these DocBlocks are generally useless.